docs/docker_compose.rst

```diff
@@ -18,7 +18,36 @@ This has been tested with docker and docker-compose versions:
   Docker version 20.10.8, build 3967b7d28e
 
   shell$ docker-compose -v
-  docker-compose version 1.27.4, build 40524192
+  docker-compose version 1.27.4, build 40524192 # version 1.27 so the deploy.replicas option works
+
+0. PreSteps
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start an AWS EC2 instance, with docker and git installed.  One needs docker-compose to run the example because EC2 does not come with docker-compose. One will also need a boto.cfg that can access a pre-existing S3 bucket.  This can be done like this (details may vary according to your AMI image):
```

Comment:
we can access S3 from a local machine as well right? Probably better to clarify Ec2 instance usage is not mandatory